fix(ai-gateway): AI provider pages review (#5758)
Kong/developer.konghq.com: app/_includes/md/ai-gateway/v2/providers.md

Reviewer: ```suggestion
> For requests with large payloads (video generation), consider increasing [`config.max_request_body_size`](/ai-gateway/entities/ai-model/#schema-aigateway-model-config-max-request-body-size) on your [AI Model](/ai-gateway/entities/ai-model/) entity to three times the raw binary size.
```

Reviewer: ```suggestion
> For requests with large payloads, consider increasing [`config.max_request_body_size`](/ai-gateway/entities/ai-model/#schema-aigateway-model-config-max-request-body-size) on your [AI Model](/ai-gateway/entities/ai-model/) entity to three times the raw binary size.
>
> Supported audio formats, voices, and parameters vary by model. Refer to your provider's documentation for available options.
```

Reviewer: ```suggestion
> For requests with large payloads, consider increasing [`config.max_request_body_size`](/ai-gateway/entities/ai-model/#schema-aigateway-model-config-max-request-body-size) on your [AI Model](/ai-gateway/entities/ai-model/) entity to three times the raw binary size.
>
> Supported image sizes and formats vary by model. Refer to your provider's documentation for allowed dimensions and requirements.
```

Reviewer: ```suggestion
{:.warning}
> Batches are configured on a separate AI Model with `type: "api"`, distinct from regular models that handle synchronous capabilities like generate and embeddings.
> Create a dedicated AI Model exclusively for batches and files, as each model must be either a regular model or an API model, not both.
{%- endif -%}
```

Reviewer: ```suggestion

{:.warning}
> Batches are configured on a separate AI Model with [`type: "api"`](/ai-gateway/entities/ai-model/#schema-aigateway-model-type), distinct from regular models that handle synchronous capabilities like generate and embeddings.
> Create a dedicated AI Model exclusively for batches and files, as each model must be either a regular model or an API model, not both.
{%- endif -%}
```

Support for {{ provider.name }}'s bidirectional streaming for realtime applications:

{:.info}

> > Realtime processing uses WebSocket protocol (ws/wss). The protocols are configured automatically when the Service and Route are created.

Reviewer: ```suggestion
> Realtime processing uses WebSocket protocol (ws/wss). This protocol is automatically enabled when you configure your [AI Model](/ai-gateway/entities/ai-model/) with the [realtime capability](/ai-gateway/entities/ai-model/#capabilities).
```
